index.html

```diff
@@ -8,10 +8,21 @@
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet" type="text/css" href="application.css">
+  <link rel='stylesheet' type='text/css' href='http://fonts.googleapis.com/css?family=Roboto'>
 </head>
 
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div name="tech" class="m-left-10">
+    <div class="tech-border">
+      <div class="tech-border tech-content">
```

Comment:
this is the only child of its parent, most of the time when this happens it means you are adding an extra unnecessary div. you can use the `tech` div for this.